pkg/types/baremetal/validation/platform_test.go

```diff
@@ -232,6 +232,12 @@ func TestValidateProvisioning(t *testing.T) {
 				BootstrapOSImage("192.168.111.1/images/qemu.x86_64.qcow2.gz?sha256=3b5a882c2af3e19d515b961855d144f293cab30190c2bdedd661af31a1fc4e2f").build(),
 			expected: "baremetal.BootstrapOSImage: Invalid value:.*: the URI provided:.*is invalid",
 		},
+		{
+			name: "invalid_clusterosimage_not_qcow2",
+			platform: platform().
+				ClusterOSImage(imagesServer.URL + "/images/metal.x86_64.gz?sha256=340dfa4d92450f2eee852ed1e2d02e3138cc68d824827ef9cf0a40a7ea2f93da").build(),
+			expected: "The URI provided:.* is an OS image and must end in .qcow2.gz or .qcow2.xz",
+		},
```

Comment:
Can we copy the test to also cover the BootstrapOSImage?

Also would be good to update the commit title e.g "Add OSImage Validation"

Reply:
yup. sounds good.